data_structures/src/error.rs

```diff
@@ -357,6 +357,18 @@ pub enum BlockError {
         proof_pkh: PublicKeyHash,
         signature_pkh: PublicKeyHash,
     },
+    /// Value Transfer Weight Overflow
+    #[fail(
+        display = "Value Transfer Transactions weight: {}, exceed the maximum weight: {}",
```

Comment:
```suggestion
        display = "Total weight of Value Transfer Transactions in a block ({}) exceeds the limit ({})",
```

Reply:
Could it be worth telling what's the offending block?

Reply:
Not needed, this is inside `BlockError` so it will be printed using something similar to `("Error when validating block {}: {}", block_hash, err)`